code/modules/clothing/masks/gasmask.dm

```diff
@@ -13,6 +13,7 @@
 	var/gas_filter_strength = 1			//For gas mask filters
 	var/list/filtered_gases = list("phoron", "sleeping_agent")
 	armor = list(melee = 0, bullet = 0, laser = 0, energy = 0, bomb = 0, bio = 75, rad = 0)
+	tint = TINT_HEAVY

```

Comment:
imho TINT_HEAVY is too much.
Its not a welding mask where you have darkened glasses.

I´d suggest trying TINT_MODERATE out.
Its what the superior welding goggles use (that the CE gets)